ci/__main__.py

```diff
@@ -0,0 +1,686 @@
+#     'examples': {
+#         'apptools',
+#         'chaco',
+#         'h5py',
+#         'numpy',
+#         'pandas',
+#         'pytables',
+#     },
+#     # Optional dependencies for some editors
+#     'editors': {
+#         'numpy',
+#         'pandas',
+#         'pillow',
+#     },
+#     # Test dependencies also applied to installation from PYPI
+#     'test': {
+#         'packaging',
+#     },
+# }
+
+# Dependencies for CI jobs using this file.
+ci_dependencies = {
+    "coverage",
+    "flake8",
+    "flake8_ets",
+}
+
+# doc_dependencies = {
+#     "sphinx",
+#     "enthought_sphinx_theme",
+#     "pyface",
+# }
+
+#: Paths to ignore in Sphinx-apidoc
+# doc_ignore = {
+#     "traitsui/qt4/*",
+#     "traitsui/wx/*",
+#     "*/tests",
+# }
+
+environment_vars = {
+    # 'pyside2': {'ETS_TOOLKIT': 'qt', 'QT_API': 'pyside2'},
+    # 'pyside6': {'ETS_TOOLKIT': 'qt', 'QT_API': 'pyside6'},
+    'pyqt5': {"ETS_TOOLKIT": "qt", "QT_API": "pyqt5"},
+    # 'pyqt6': {"ETS_TOOLKIT": "qt", "QT_API": "pyqt6"},
+    # 'wx': {'ETS_TOOLKIT': 'wx'},
+    # 'null': {'ETS_TOOLKIT': 'null'},
+}
+
+# Location of documentation files
+HERE = os.path.dirname(__file__)
+DOCS_DIR = os.path.join(HERE, "docs")
+
+# Location of news fragment for creating changelog.
+NEWS_FRAGMENT_DIR = os.path.join(DOCS_DIR, "releases", "upcoming")
+
+# Location of the Changelog file.
+CHANGELOG_PATH = os.path.join(HERE, "CHANGES.txt")
+
+
+def normalize(name):
+    return name.replace("_", "-")
+
+
+# Ensure that "-h" is supported for getting help.
+CONTEXT_SETTINGS = dict(
+    help_option_names=["-h", "--help"],
+    token_normalize_func=normalize,
+)
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+def cli():
+    pass
+
+
+@cli.command()
+@click.option('--runtime', default=DEFAULT_RUNTIME)
+@click.option('--toolkit', default=DEFAULT_TOOLKIT)
+@click.option('--environment', default=None)
+@click.option(
+    "--editable/--not-editable",
+    default=False,
+    help="Install main package in 'editable' mode?  [default: --not-editable]",
+)
+@click.option('--source/--no-source', default=False)
+def install(runtime, toolkit, environment, editable, source):
+    """Install project and dependencies into a clean EDM environment."""
+    parameters = get_parameters(runtime, toolkit, environment)
+    packages = (dependencies | ci_dependencies)
+
+    # edm commands to setup the development environment
+    commands = [
+        "edm environments create {environment} --force --version={runtime}"
+    ]
+
+    commands.extend(
+        [
+            "edm install -y -e {environment} " + " ".join(packages),
+            # "edm run -e {environment} -- pip install --force-reinstall -r
+            # ci-src-requirements.txt --no-dependencies",
+            "edm run -e {environment} -- python setup.py clean --all",
+        ]
+    )
+
+    # pip install pyqt5, pyqt6, pyside2 and pyside6, because we don't have them
+    # in EDM yet
+    if toolkit == 'pyside2':
```

Comment:
`pyside2` is now available in EDM, or at least version 5.14.

Reply:
Ok, will remove this special consideration.